### PR TITLE
Refactor: <SignMessageButton /> [WLT-4710]

### PR DESCRIPTION
**Pull request by everdimension**

### CODE REVIEW

File: src/ui/pages/HardwareWalletConnection/HardwareSignMessage/HardwareSignMessage.tsx [hunk 1]
```diff
@@ -14,151 +14,145 @@ import { hardwareMessageHandler } from '../shared/messageHandler';
...
+            disabled={isLoading || isSigning}
+            style={{
+              paddingInline: 16, // fit longer button label
+            }}
+            {...buttonProps}
+          >
+            <HStack gap={8} alignItems="center" justifyContent="center">
+              <LedgerIcon />
+              {children ||
+                (isSigning ? 'Sending' : buttonTitle || 'Sign with Ledger')}
```
Comment (zerts) [inline src/ui/pages/HardwareWalletConnection/HardwareSignMessage/HardwareSignMessage.tsx:151]: Did we decide not to use ellipsis here?

File: src/ui/pages/HardwareWalletConnection/HardwareSignMessage/HardwareSignMessage.tsx [hunk 2]
```diff
@@ -14,151 +14,145 @@ import { hardwareMessageHandler } from '../shared/messageHandler';
...
+            title="Follow instructions on your ledger device"
+          >
+            <TextPulse>Sign on Device</TextPulse>
+          </Button>
+        ) : (
+          <Button
+            kind={buttonKind}
+            disabled={isLoading || isSigning}
+            style={{
+              paddingInline: 16, // fit longer button label
```
Comment (zerts) [inline src/ui/pages/HardwareWalletConnection/HardwareSignMessage/HardwareSignMessage.tsx:144]: Can be written in 1 line, I suppose :)

**Review by zerts — APPROVED** [review]

File: src/ui/pages/HardwareWalletConnection/HardwareSignMessage/HardwareSignMessage.tsx (hunk 1 above)
Comment (everdimension) [reply src/ui/pages/HardwareWalletConnection/HardwareSignMessage/HardwareSignMessage.tsx:151]: For some reason maybe 🤷‍♀️

File: src/ui/pages/HardwareWalletConnection/HardwareSignMessage/HardwareSignMessage.tsx (hunk 2 above)
Comment (everdimension) [reply src/ui/pages/HardwareWalletConnection/HardwareSignMessage/HardwareSignMessage.tsx:144]: yeah, but the comment would be moved out then ¯\\\_(ツ)_/¯

**Pull request merged**